.github/workflows/helm-lint.yaml

```diff
@@ -100,7 +100,7 @@ jobs:
         # default value for event_name != workflow_dispatch
       - name: Run helm upgrade
         run: |
-          helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
-          helm install sdf tractusx-dev/sdfactory
-          helm upgrade sdf charts/sdfactory
+          helm repo add tractusx https://eclipse-tractusx.github.io/charts/dev
+          helm install sdf tractusx/sdfactory --version ${{ github.event.inputs.upgrade_from || '2.1.7' }}
+          helm upgrade sdf tractusx/sdfactory
```

Comment:
You should use `charts/sdfactory` for upgrade, not the public helm repository. This workflow is designed to test even unreleased helm charts, while opening PRs. You setup would require the have the "upgrade to" helm chart version to be available in public helm chart repo.

Reply:
Hi @carslen : I have updated it.